deepchem/models/torch_models/oneformer.py

```diff
@@ -0,0 +1,459 @@
+from PIL import Image
+import torch
+import numpy as np
+
+import time
+import logging
+from collections.abc import Sequence as SequenceCollection
+from typing import Union, Dict, Tuple, Iterable, List, Optional, Callable, Any
+
+from deepchem.models.optimizers import LearningRateSchedule
+from deepchem.trans import Transformer, undo_transforms
+from deepchem.utils.typing import LossFn, OneOrMany
+from deepchem.models.torch_models import HuggingFaceModel
+
+from transformers import AutoProcessor, AutoModelForUniversalSegmentation, OneFormerConfig
+
+logger = logging.getLogger(__name__)
```

Comment:
please add the torch and hf imports under a try-except block, this should resolve the test failuers